### ISSUE TITLE
[Bug Report][3.6.0] VSelect menu shifted few pixels to the right

### ISSUE THREAD

**Issue by crystalfp** [vuetifyjs/vuetify#19732]
### Environment
**Vuetify Version:** 3.6.0
**Vue Version:** 3.4.26
**Browsers:** Chrome 124.0.0.0
**OS:** Windows 10

### Steps to reproduce
Open the dropdown menu.
Notice the menu is shifted to the right by 5-10 pixels.
Now remove the width style on the container div and the menu returns to normal position.

### Expected Behavior
The menu be exactly below the selector.

### Actual Behavior
It is shifted to the right if the container specifies a width.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqlkcFugzAMhl8lyoULDC67IDZpqrS9wKQdSg8e8VpLJomSwIqmvfsCTCXttZfot/M5sf3vf6R3Xfli7cM4oKxlE7C3DAGfWy1EMxZg7SJjoGj8l2sgfJgYn1r5TSqcavFYVfbcyguz1HdGByCNTnQM3kfaQlFdUQvnkbELaVKImmI3c8k+2wHTl3GaIMtFtjNsHCgz69coSS3pNzTuuBLveAY/i4/J9KSPWX7/E4dWXvc3giPQITY4aIWO45jqllGoPYUpMp3p4/cBPhlvoRMpLBTGRbFPb8p0l2WyzM2HcnNl0zO7GteUiaHyN9cD83Ic/gDpPJZ/)

### Other comments
The two <div> containers are part of the application.

**Comment by johnleider** [comment]
This is happening because of edge detection and I don't think it's a bug. @KaelWD thoughts?

**Comment by crystalfp** [comment]
Respectfully disagree. An unexpected behavior caused by something outside my control that gives a visual look not in line with the quality of the framework for me is a bug.

Anyway, changing the DIVs with VContainers changed nothing.
The width cannot be removed because the selector is in a side panel of a flex container.

Thanks for looking!
mario

**Comment by johnleider** [comment]
I'm not saying your issue doesn't have merit, just that it's not a bug. If it's not something that can be disabled maybe we add an option to do just that.

**Comment by yuwu9145** [comment]
locationStrategy has a hardcoded viewportMargin as 12px, seems to deliberately keep Overlay minimum 12px away from edge by default. There must be a good reason behind it.

If parent div has a margin over 12px, it's working as expected [demo](https://play.vuetifyjs.com/#eNqlUUFOwzAQ/IrlSy4NCUhcQkBCleADSByaHpZ4aVfa2JbthESIv+O4iKS99mLNrmfX45ndt/SuLZ6tvRl6lJWsA3aWIeBTo4WohxysTTAWioY/eCqEDxPjYyO/SIVjJe7L0o4PogN3IF2J2zs7NvJ/Im1rjQ5AGp1oGbyPsxby8oyVeB4Z27BuClFR1DaP7LItMH0apwmyjci2ho0DZWb8EiGp1H5FE4Uk+IYj+Bm8T6Yjfcg216/YN/Jc3wCOQIcosNcKHcdvqkuOQu0pTJHTmi4+H+CD8ZJ0JIW5wmgU+/VNsfayWJm5pFIsGS145p5irItVvPJno3vmdOx/AepBmv0=)

**Comment by crystalfp** [comment]
Yes, putting the margin makes the dropdown stay where it should stay. Well, the overall effect is not nice as before (the selector filling the upper side of the sidebar) but acceptable.
Will be nice if you would like to investigate if this margin is really necessary o if it is possible to switch off (maybe with an option).
Anyway, thanks a lot for vuetify! With it my application has a very professional look.

**Comment by johnleider** [comment]
I think that the user should be able to opt out of this functionality. Surely it can't be too difficult to allow disabling.

**Comment by yuwu9145** [comment]
@KaelWD What was the reason behind https://github.com/vuetifyjs/vuetify/blob/767ecdcedd75a6d28379ee4058b8c3b70bb66feb/packages/vuetify/src/components/VOverlay/locationStrategies.ts#L235

**Comment by KaelWD** [comment]
```html
<template>
  <v-app>
    <v-navigation-drawer>
      <v-select />
    </v-navigation-drawer>
    <v-container>
      <div style="height: 1000px" />
      <v-btn>
        <v-menu activator="parent" location="end">
          <v-list>
            <v-list-item v-for="i in 100">Item</v-list-item>
          </v-list>
        </v-menu>
      </v-btn>
      <div style="height: 3000px" />
    </v-container>
  </v-app>
</template>
```

**Comment by Fengtao1314520** [comment]
I also found this issue, when v-select in v-navigation-drawer, was shifted to the right by 12 pixels, and using dev tools to found that .v-overlay-container .v-combobox__content , that left is 12 PX. i had to change global to modify this issue. but ti still modify other v-select too, please consider this scenario, it's very helpful for me.

and it's a demo [LINK
](https://play.vuetifyjs.com/#eNqNVE1v2zAM/SuEL3aAOMm6W5AEGwKs2LCPQwfs0PSg2rSjVZYMSc4cBPnvoyx/ptjQi21afOTjI6nHS2B0svxYlotThcE62FgsSsEs7g4SYHOKBTuryoLGbHsIWFkeAkgEM4YsrSqZYgrtOy7SQ9DAGiD5xs9MQ6KE0uSdazzHgudHizK+ozCSFehjOr8eSuDkyEUKp9gIZcnjAqXm0sJ15NOkeLayJ1PUMausorgfEsGTF/rVoKKwTRDOCH6PFlJm2WbZoIeUyyZnx55OWxT96SuS7MRzZrmScarZH7ypLWX6ZVqa9yK7RF0widIOCVN+6rmncSawht+VsTw7xwk5UnBGYsnOOMbvVqtp/W9SqdfpVhXP7b+i3MribKI9WN2stCTWpBnKdIDvKLdBgYkdExLsGQURefAnwfhszSmmk+Qx3FP9mdKSs3AO4d4pzVLlvj/RJ0+b3/eodO49fmLNjPv4dVYFl3n4NA19Ypoz6ZQyFGt8tnOFe56jSieL4Gfi1QCMp6NgvDXgtrGTTk67PPAw9izc0BD3+IhuT9bwfrUq686l5/aNMsFeEVy2wjpyLv+YT6aUS9eOordoFKlHQ5H/WqFBK4s1tejV9PhwzfQMOnZzNGg4nidneZi/Wsj0dwuZE7XJNInmpW8GL0qlLVygMvjV30VXyLQqIKQbyykZ+qITJY2FfbMUW7j4pAZtVUalVqWZQ2LrWXfQ+V8gR+vjfiYOFHs7ZIpmnZ4AWSUT13m/ZdELnkexgFqM2kZfHn58XxhLHjkRiyahG8gcZCXEHO5mFNojr0MOTWy1hGgG251ju3BbZRYpZqwSNuo3vMVe5+7d4rFudGp9O2qJIvncxWPWVKoXp4dtlr3OwXXeEHOPp796yO6m)

**Comment by crystalfp** [comment]
Problem still in vuetify 3.7.1

**Comment by crystalfp** [comment]
Issue still here in 3.7.5
Any news?

**Comment by rsaleri** [comment]
It's really annoying, I'm rewriting an application that uses a lot of v-select adjacent to the right edge. Vuetify 1 (the current version of the application is quite old) didin't have this issue and it might be an issue.

**Comment by crystalfp** [comment]
I'm confused.
The shift of 12 px is still here in vuetify 3.9.5
Adding `stick-to-target` to v-select changes nothing.
Please enlighten me.
Thanks!

**Comment by J-Sek** [comment]
Merged to `dev` means it goes out in 3.10.0. You could point to @vuetify/nightly

```diff
-"vuetify": "3.9.5"
+"vuetify": "npm:@vuetify/nightly@3.9.5-dev.2025-08-12"
```

...but I don't recommend it yet. The prop applies to all VMenu's by default and has a minor but annoying bug we still have to fix.
